### ISSUE TITLE
Per-entity theming does not affect top-of-the-page badges on 0.85.*

### ISSUE THREAD

**Issue by sendorm** [andrey-git/home-assistant-custom-ui#166]
### Checklist
- [x] I'm running the latest version of CustomUI ([Update guide](https://github.com/andrey-git/home-assistant-custom-ui/blob/master/docs/installing.md#updating)) or using a specific [release](https://github.com/andrey-git/home-assistant-custom-ui/releases) that is not marked as "Broken".
- [x] I tried to force-refresh (Ctrl+Shift+R / Ctrl+F5) the browser
- [x] (Optional, but recommended) I'm using Chrome or tried to reproduce the feature on Chrome.

**Browser + Version:**  Opera 57 and iOS 12 

**CustomUI version:** 20190113

**Home Assistant release (`hass --version`):** 0.85.1 

The theming for top of the page badges were working on 84.6 but not anymore on 85.1.

**Comment by Mariusthvdb** [comment]
maybe you should rename this to:

Broken: theming to top of the page badges.

in my case even with badges created in python, which were always exempt of Hassio quirks...

still: please bring back theming for top of the page badges:

HA 84.3
![Schermafbeelding 2019-04-03 om 14 19 22](https://user-images.githubusercontent.com/33354141/55478263-865f2980-561b-11e9-9159-9ad9eceb2f8e.jpg)

HA 90.2
![Schermafbeelding 2019-04-03 om 14 19 28](https://user-images.githubusercontent.com/33354141/55478264-86f7c000-561b-11e9-8842-cc6fa2854f11.jpg)

**Comment by sendorm** [comment]
I don’t think it will ever be done :( Changed most of my entries to bar-card.

**Comment by Mariusthvdb** [comment]
full example:
```
sensor:
  - platform: template
    sensors:
      levering_of_verbruik:
        friendly_name_template: >
          {% if states('sensor.netto_verbruik')|int > 0 %} Verbruik
          {% else %} Levering
          {% endif %}
        icon_template: >
          {% if states('sensor.netto_verbruik')|int > 0 %} mdi:import
          {% else %} mdi:export
          {% endif %}
        value_template: >
          {{states('sensor.netto_verbruik')}}
```

customizes like:

```
homeassistant:
  customize:

    sensor.levering_of_verbruik:
      state_card_mode: badges
      templates:
    #    icon_color: >
    #      if (state === 'Levering') return 'rgb(251, 210, 41)';
    #      return 'rgb(54, 95, 140)';
        theme: >
          if (state > 0) return 'orange';
          return 'green';
        unit_of_measurement: >
          ${Math.round(entities['sensor.calculated_bruto_verbruik'].state)}
```

Ha 84.3

![Schermafbeelding 2019-04-03 om 14 36 23](https://user-images.githubusercontent.com/33354141/55479353-30d84c00-561e-11e9-9e1c-bc9f6b4843ef.jpg)

HA 90.2

![Schermafbeelding 2019-04-03 om 14 36 17](https://user-images.githubusercontent.com/33354141/55479374-3d5ca480-561e-11e9-9daa-e416de933b7c.jpg)

**Comment by horacecar** [comment]
hello, does anyone know if there's any update on this ??

**Comment by eon123** [comment]
Any updates on this? thanks.

**Comment by LupoCri** [comment]
Hi. 
Any updates?
Tnx.

**Comment by sendorm** [comment]
I’ve stopped using badges long time ago. Still have no idea why this was removed from HA.

**Comment by Mariusthvdb** [comment]
this  whole repo is no longer supported, not does it work correctly in current Lovelace. A reduced and updated (hacked) version is available `https://github.com/Mariusthvdb/custom-ui/`

theming is not supported either, but you can use the card-mod styling for that https://community.home-assistant.io/t/card-mod-add-css-styles-to-any-lovelace-card/120744/1308

**Comment by LupoCri** [comment]
Tnx
